Formula/at-spi2-core.rb

```diff
@@ -10,11 +10,10 @@ class AtSpi2Core < Formula
   end
 
   depends_on "gobject-introspection" => :build
-  depends_on "intltool" => :build
   depends_on "meson" => :build
   depends_on "ninja" => :build
   depends_on "pkg-config" => :build
-  depends_on "python@3.10" => :build
+  depends_on "python@3.11" => :build
   depends_on "dbus"
   depends_on "gettext"
   depends_on "glib"
```

Comment:
Could be better to wait for alias migration to avoid having an indirect dependency on `python@3.10`

Reply:
On the other hand this is a build dependency, so I would not bother that much. I'm ok with that change.